### PR TITLE
Refactor/Feature: Move interfaces to separate module  & use github actions

### PR DESCRIPTION
**Pull request by brikendr**
- [x] Refactor code to use the @huddly/sdk-interfaces repo
- [x] Update documentation

### CODE REVIEW

File: src/components/device/ace.ts [hunk 1]
```diff
@@ -42,8 +43,7 @@ export default class Ace implements IIpDeviceManager, IUVCControls {
 
   private readonly GPRC_CONNECT_TIMEOUT: number = 1; // seconds
   private readonly GRPC_PORT: number = 50051;
-  private devMode: boolean = false;
-  private devModeGrpcClient?: HuddlyServiceClient;
+  private huddlyGrpcClient: HuddlyServiceClient;
```
Comment (eskilgh) [inline src/components/device/ace.ts:46]: nice

**Review by eskilgh — APPROVED** [review]
This looks good. My only worry is how this will affect the documentation pipeline.

**Comment by brikendr** [comment]
> This looks good. My only worry is how this will affect the documentation pipeline.

I will make sure to update the documentation accordingly. I have to do it in this PR since the site developer.huddly.com is update automatically when we merge into master

**Review by pkhansen — APPROVED** [review]
LGTM! @ignore is a nice addition  :)

**Pull request merged**